### PR TITLE
add coturn to synapse

### PR DESCRIPTION
**Pull request by n0emis**
this allows users on this matrix-hs to participate in (video)-calls

### CODE REVIEW

**Review by leona-ya — CHANGES_REQUESTED** [review]
I think the port allow-rule in firewalld is missing (add to `firewall_ports` variable)

**Review by leona-ya — APPROVED** [review]
LGTM, but I would like to wait until the conference on next week Sunday with merging

File: group_vars/all.yml [hunk 1]
```diff
@@ -73,8 +73,18 @@ synapse_federation_listener_port: '86'
 synapse_app_service_config_files:
   - /etc/matrix-appservice-irc/appservice-registration-irc.yaml
 
+synapse_turn_uris:
+  - "turn:turn.chaos.jetzt:3479?transport=udp"
```
Comment (e1mo) [inline group_vars/all.yml:77]: Please use variables

File: group_vars/all.yml [hunk 2]
```diff
@@ -73,8 +73,18 @@ synapse_federation_listener_port: '86'
...
+  - "turn:turn.chaos.jetzt:3479?transport=tcp"
```
Comment (e1mo) [inline group_vars/all.yml:78]: Same here

File: group_vars/all.yml [hunk 3]
```diff
@@ -73,8 +73,18 @@ synapse_federation_listener_port: '86'
...
+
+synapse_turn_shared_secret: "{{ coturn_static_auth_secret }}"
+synapse_turn_allow_guests: True
+
+coturn_static_auth_secret: "{{ lookup('passwordstore', 'infra/coturn') }}"
+coturn_realm: "turn.chaos.jetzt"
```
Comment (e1mo) [inline group_vars/all.yml:84]: And here

File: group_vars/all.yml [hunk 4]
```diff
@@ -295,7 +305,7 @@ mumble_web_listen: "84"
 mumble_web_ssl_activated: False
 mumble_web_ssl_target: True
 
-traefik_version: "v2.2.1"
+traefik_version: "v2.2.8"
```
Comment (e1mo) [inline group_vars/all.yml:308]: Same goes here

File: group_vars/all.yml [hunk 5]
```diff
@@ -73,8 +73,18 @@ synapse_federation_listener_port: '86'
...
+
+synapse_turn_shared_secret: "{{ coturn_static_auth_secret }}"
+synapse_turn_allow_guests: True
+
+coturn_static_auth_secret: "{{ lookup('passwordstore', 'infra/coturn') }}"
+coturn_realm: "turn.chaos.jetzt"
+
 riot_installation_path: "/var/www/riot-web"
-riot_version: "1.6.8"
+riot_version: "1.7.2"
```
Comment (e1mo) [inline group_vars/all.yml:87]: I would not mix multiple topics in one PR

File: group_vars/all.yml [hunk 6]
```diff
@@ -55,7 +55,7 @@ synapse_enable_group_creation: true
 synapse_enable_registration: true
 synapse_domain: "{{ base_url }}"
 synapse_server_fqdn_matrix: "matrix.{{ synapse_domain }}"
-synapse_pip_version: "1.16.0"
+synapse_pip_version: "1.18.0"
```
Comment (e1mo) [inline group_vars/all.yml:58]: Is it required for calls? Otherwise I would not mix multiple topics in one PR

**Review by e1mo — CHANGES_REQUESTED** [review]

**Pull request closed without merging**